docker/spec/fixtures/helm/yaml/double-quotes.yaml

```diff
@@ -0,0 +1,4 @@
+image:
+  repository: 'nginx'
+  tag: "1.14.2"
+  registry: registry.example.com
```

Comment:
Why not make all of these strings double-quoted? ie, `nginx` and `registry.example.com` could be double-quoted strings too...

Reply:
Done